bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -63,4 +63,5 @@ apiServerArguments:
     - "NodeDisruptionExclusion=true"
     - "ServiceNodeExclusion=true"
     - "SCTPSupport=true"
+    - "IPv6DualStack=true"
```

Comment:
@deads2k i guess we have to remove this as well (in case multus connects to bootstrap apiserver...)